packages/v4/patternfly-docs/content/accessibility/avatar/avatar.md

```diff
@@ -3,21 +3,54 @@ id: Avatar
 section: components
 ---
 
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+
 An **avatar** is a visual used to represent a user. It may contain an image or a placeholder graphic. Typical usage is to represent the current user in the masthead.
 
-**Keyboard users** shouldn’t be able to interact with the avatar on its own. Thus, the user should not be able to focus on the avatar using **Tab** to move forward and **Tab + Shift** to move backwards through interactive elements.
+## Accessibility
 
-**Screen reader users** should be able to navigate to the avatar and it should have alternative text, since it’s an image element. You can use the `alt` prop to provide alternative text.
+To implement an accessible PatternFly **avatar**:
 
-If you’re combining an avatar with another component, make sure to check accessibility guidelines for that component as well.
+- Use the `alt` React prop to provide alternative text for the avatar image.
+- If you are using an SVG element for the avatar, [learn how to create accessible SVGs](https://www.deque.com/blog/creating-accessible-svgs/) and reference the different SVG patterns. Screen reader accessibility for SVGs varies based on the pattern being used.
```

Comment:
@thatblindgeye which text do you think it should be? The first time it's written into the sentence, but I used the page title the second time, since it's not part of a sentence.

Reply:
I'd personally lean more towards the page title like you did in the second instance of it